src/plugins/intel_cpu/src/utils/ngraph_utils.hpp

```diff
@@ -50,23 +50,7 @@ inline bool isDynamicNgraphNode(const std::shared_ptr<const ov::Node>& op) {
 }
 
 inline std::string get_port_name(const ov::Output<const ov::Node>& port) {
-    std::string name;
-    // Should use tensor name as the port name, but many legacy tests still use legacy name
-    // plus sometimes it will get empty tensor name.
-    const bool is_legacy_api = false;
-    if (!is_legacy_api) {
-        // TODO: To apply unified tensor name.
-    }
-    if (name.empty()) {
-        bool is_input = ov::op::util::is_parameter(port.get_node());
-        if (is_input) {
-            name = ov::op::util::get_ie_output_name(port);
-        } else {
-            const auto node = port.get_node_shared_ptr();
-            name = ov::op::util::get_ie_output_name(node->input_value(0));
-        }
-    }
-    return name;
+    return port.get_node_shared_ptr()->get_friendly_name();
```

Comment:
we should not use names at all, because it's not reliable and can be changed (mistakenly) by transformations

Reply:
Per deprecated message, suggest applying `ov::ISyncInferRequest::find_port`

https://github.com/openvinotoolkit/openvino/blob/0ef2fab349036dd4bbabf6771647d14379063ff9/src/common/transformations/include/transformations/utils/utils.hpp#L89